README.md

```diff
@@ -210,13 +210,27 @@ Functional tests live in [server-syncstorage](https://github.com/mozilla-service
 
 ## Creating Releases
 
-Open a PR after doing the following:
-
-1. Bump the version number in [Cargo.toml](https://github.com/mozilla-services/syncstorage-rs/blob/master/Cargo.toml).
-2. `cargo build --release` - Build with the release profile [release mode](https://doc.rust-lang.org/book/ch14-01-release-profiles.html).
-3. `clog -C CHANGELOG.md` - Generate release notes. We're using [clog](https://github.com/clog-tool/clog-cli) for release notes. Add a `-p`, `-m` or `-M` flag to denote major/minor/patch version, ie `clog -C CHANGELOG.md -p`.
-
-Once your PR merges, then go ahead and create an official [GitHub release](https://github.com/mozilla-services/syncstorage-rs/releases).
+1. Switch to master branch of syncstorage-rs
+2. `git pull` to ensure that the local copy is up-to-date.
+3. `git pull origin master` to make sure that you've incorporated any changes to the master branch.
+4. `git diff origin/master` to ensure that there are no local staged or uncommited changes.
+5. Bump the version number in [Cargo.toml](https://github.com/mozilla-services/syncstorage-rs/blob/master/Cargo.toml) (this new version number will be designated as `<version>` in this checklist)
+6. create a git branch for the new version `git checkout -b release/<version>`
+7. `cargo build --release` - Build with the release profile [release mode](https://doc.rust-lang.org/book/ch14-01-release-profiles.html).
+8. `clog -C CHANGELOG.md` - Generate release notes. We're using [clog](https://github.com/clog-tool/clog-cli) for release notes. Add a `-p`, `-m` or `-M` flag to denote major/minor/patch version, ie `clog -C CHANGELOG.md -p`.
```

Comment:
I do clog `--setversion <version>`, it's a little more "automatic" than clog's bumping flags (I didn't even know those existed)

Reply:
Yeah, FWIW, I do it manually as well.